hazelcast/src/main/java/com/hazelcast/internal/networking/nio/NioOutboundPipeline.java

```diff
@@ -137,8 +143,8 @@ private long idleTimeMs() {
     }
 
     @Probe(level = DEBUG)
-    private long isScheduled() {
-        return scheduled.get() ? 1 : 0;
+    private long scheduled() {
+        return scheduled.get().ordinal();
```

Comment:
This is rather unconventional. Any reason we can't use something like `return scheduled.get() == State.SCHEDULED ? 1 : 0`?
I'm thinking if someone changes the order. The value of this method returns as well.